CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## v3.0.0
```

Comment:
* Delete mention of migration_to_v2_0_tests from tests/README.md
* Delete these two lines from `.gitignore`:  
  ```
   # only because of migration_to_v2_0_tests tests
   /Pipfile*
  ```